source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -910,7 +924,7 @@ namespace RendererQml
         }
         else
         {
-            int imageWidth;
+            int imageWidth = 0;
```

Comment:
seems like this is not changing can we make it const

Reply:
We are changing this in the switch statement